### PR TITLE
Userbot disable feature #213

### PR DESCRIPTION
**Pull request by Krishna-Singhal**

### CODE REVIEW

**Comment by rking32** [comment]
nice

File: userge/plugins/tools/system.py [hunk 1]
```diff
@@ -29,6 +31,14 @@ async def _init() -> None:
     if d_s:
         Config.RUN_DYNO_SAVER = bool(d_s['on'])
         MAX_IDLE_TIME = int(d_s['timeout'])
+    disabled_all = await SAVED_SETTINGS.find_one({'_id': 'DISABLED_CHATS'})
```
Comment (rking32) [inline userge/plugins/tools/system.py:34]: `DISABLED_CHATS` confusing. can you change it to like `DISABLE_ALL_CHATS` something

File: userge/plugins/tools/system.py [hunk 2]
```diff
@@ -179,6 +189,96 @@ async def getvar_(message: Message) -> None:
...
+                message.edit(
+                    f"CHAT : `{chat.title}` removed from **DISABLED_CHATS**!",
+                    del_in=5,
+                    log=__name__
+                )
+            )
+
+
+@userge.on_cmd("dishere", about={
+    'header': "enable userbot in current chat.",
```
Comment (rking32) [inline userge/plugins/tools/system.py:234]: what ? `enable userbot in current chat.`

Comment (Krishna-Singhal) [reply userge/plugins/tools/system.py:234]: 😅😅

**Pull request merged**